config-templates/shopify/order_line.yaml

```
@@ -0,0 +1,40 @@
+# ORDER_LINE CONFIG
+data_source:
+  name: shopify_order_line
+  owners:  # This will throw an error unless you add a list of owners or remove this entry.
+  sql_query: |
+    select * from SHOPIFY.ORDER_LINE a left join SHOPIFY."ORDER" b on a.order = b.order # We're joining because any data source with measures needs a primary time dimension. 
```

Does this comment syntax work in the multi-line entry case? It's not highlighting correctly and I'm still kind of a YAML novice, so I don't know how the parser treats it.

In any case, it might be clearer to put the comment on its own line, something like:

```suggestion
  # We're joining because any data source with measures needs a primary time dimension.
  sql_query: |
    select * from SHOPIFY.ORDER_LINE a left join SHOPIFY."ORDER" b on a.order = b.order
```